spec/controllers/case_logs_controller_spec.rb

```diff
@@ -162,10 +162,10 @@
 
       it "routes to the appropriate conditional page based on the question answer of the current page" do
         post :submit_form, params: { id: id, case_log: case_log_form_conditional_question_yes_params }
-        expect(response).to redirect_to("/case_logs/#{id}/conditional_question_yes_page")
```

Comment:
Shouldn't all the pages be dashes as well? i.e. `"/case_logs/#{id}/conditional-question-yes-page"`